soc/riscv/riscv-ite/common/chip_chipregs.h

```diff
@@ -1869,6 +1869,24 @@ struct gctrl_it8xxx2_regs {
 /* bit[2] = 1: Enable global reset. */
 #define IT8XXX2_GCTRL_GRST		BIT(2)
 
+/**
+ *
+ * (22xxh) Battery-backed SRAM (BRAM) registers
+ *
+ */
+#ifndef __ASSEMBLER__
+/* Battery backed RAM indices. */
+enum bram_indices {
+
+	/* This field is used to indicate BRAM is valid or not. */
+	BRAM_IDX_VALID_FLAGS0  = 0xbc,
+	BRAM_IDX_VALID_FLAGS1  = 0xbd,
+	BRAM_IDX_VALID_FLAGS2  = 0xbe,
+	BRAM_IDX_VALID_FLAGS3  = 0xbf
+
+};
```

Comment:
I would prefer to remove the magic numbers in the code.

#define BRAM_MAGIC_FIELD_OFFSET 0xbc
enum bram_indices {
        /* This field is used to indicate BRAM is valid or not. */
        BRAM_IDX_VALID_FLAGS0  = BRAM_MAGIC_FIELD_OFFSET,
        BRAM_IDX_VALID_FLAGS1,
        BRAM_IDX_VALID_FLAGS2,
        BRAM_IDX_VALID_FLAGS3
};

Reply:
Done.